backend/src/main/java/com/agh/introwertycznelosie/data/RecruitmentStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.agh.introwertycznelosie.data;
+
+public enum RecruitmentStatus {
+    open, closed
```

Comment:
a nie powinno byc jeszcze w trakcie?

Reply:
open jest w trakcie, closed zakończony, jak chce się zrobić reopen to się zmienia na open, przy tworzeniu defaultowo jest zamknięty